### ISSUE TITLE
Overhead filter in Users page 

### ISSUE THREAD

**Issue by RickieT** [mercycorps/TolaActivity#1518]
Referencing #1263 - The overhead filter has changed a little bit as far as how the styling works, now when making multiple selections, it counts as "2 selected" when selecting more than one, but when you select a single program, its a little wonky, and the x appears below a selection:
![Screen Shot 2019-04-29 at 9 52 24 AM](https://user-images.githubusercontent.com/23220180/56912746-cec80680-6a64-11e9-9406-d50fab6ca873.png)
![Screen Shot 2019-04-29 at 9 52 15 AM](https://user-images.githubusercontent.com/23220180/56912747-cec80680-6a64-11e9-96cf-4d47c4c83a70.png)

**Comment by jennyhavah** [comment]
@axoplasm I'm not sure what the original intention was, or how it used to work, will defer to you on what exactly/when to fix.

**Comment by axoplasm** [comment]
This is a multiselect. This should have been caught when we de-pilled the multiselects, or maybe this only happens when we have just one option (?) 

This close to deployment I’m tempted to defer.

**Comment by RickieT** [comment]
> This is a multiselect. This should have been caught when we de-pilled the multiselects, or maybe this only happens when we have just one option (?)
> 
> This close to deployment I’m tempted to defer.

Yeah it looks like this one got missed, but the criteria for what fields and forms were getting this treatment was vague to begin with.
